doc/release-notes.md

```diff
@@ -74,6 +74,15 @@ P2P and network changes
   node using P2P relay. This version reduces the initial broadcast guarantees
   for wallet transactions submitted via P2P to a node running the wallet. (#18038)
 
+- The Tor hidden service that is automatically created (`-listenonion`) will now
+  be created as a Tor v3 service instead of Tor v2. The private key that was used
+  for Tor v2 (if any) will be left untouched in `onion_private_key` in `-datadir`
+  and can be removed if not needed. Bitcoin Core will no longer attempt to read it.
+  The private key for the Tor v3 service will be saved in `onion_private_key_torv3`.
+  To use the deprecated Tor v2 service (not recommended), then `onion_private_key`
+  can be copied over `onion_private_key_torv3`
+  (`cp -f onion_private_key onion_private_key_torv3`).
```

Comment:
(`code`) markup inside parentheses; maybe easier to read as: 
```suggestion
  , e.g. `cp -f onion_private_key onion_private_key_torv3`.
```

Reply:
Done